tests/deployments/puppet/puppet_test.py

```diff
@@ -263,6 +263,13 @@ def test_puppet_on_windows(puppet_version):
             assert wait_for(
                 p(has_datapoint_with_metric_name, backend, "sfxagent.datapoints_sent")
             ), "Didn't get internal metric datapoints"
+
+            # re-apply without any changes
+            run_win_puppet_agent(backend, monitors, INITIAL_VERSION, STAGE)
+            backend.reset_datapoints()
+            assert wait_for(
```

Comment:
Would this have failed without the updates to the manifest?

Reply:
It shouldn't have failed. I just wanted to add another scenario to the test to make sure that everything is still running after applying without changes.